src/main/scala/mesosphere/marathon/core/deployment/impl/TaskReplaceActor.scala

```diff
@@ -153,6 +153,44 @@ class TaskReplaceActor(
 
   }
 
+  private def launching: Receive = {
+    case Frame(instances) =>
```

Comment:
This is just a prototype idea. There some code duplication to `launchInstances`. @timcharper, how would you manipulate the frame? I don't like that I call `scheduler.reschedule` etc.